### PR TITLE
fix: design fixes for table component

### PR DESCRIPTION
**Pull request by dcshzj**
Split from #74, only for table component.

Fixes ISOM-964.

### CODE REVIEW

**Comment by dcshzj** [comment]
!run chromatic

File: src/templates/next/components/internal/Notification/Notification.tsx [hunk 1]
```diff
@@ -14,7 +14,10 @@ const NotificationBanner = ({ content }: NotificationProps) => {
   return (
     isShown && (
       <div className="flex flex-col sm:flex-row justify-between gap-6 bg-site-primary-100 py-4 px-5 sm:px-10">
-        <BaseParagraph className="text-base" content={content}></BaseParagraph>
+        <BaseParagraph
+          className="text-base text-content"
+          content={content}
+        ></BaseParagraph>
```
Comment (alexanderleegs) [inline src/templates/next/components/internal/Notification/Notification.tsx:20]: Assuming that this was intended to go in for a different issue!

File: src/templates/next/components/native/Table/Table.tsx [hunk 2]
```diff
@@ -6,16 +6,39 @@ import BaseParagraph from "../../internal/BaseParagraph"
...
+      if (!stickyRowIndexes.includes(index)) {
+        return
+      }
+
+      if (cell.rowSpan) {
+        stickyRowIndexes = stickyRowIndexes.filter(
+          (value) => value <= index || value >= index + cell.rowSpan!,
+        )
+      }
+    })
```
Comment (alexanderleegs) [inline src/templates/next/components/native/Table/Table.tsx:33]: Could you add a comment explaining what this section is doing? It's a little complicated haha

File: src/interfaces/native/Text.ts [hunk 3]
```diff
@@ -43,6 +43,6 @@ export type Marks =
 
 export interface TextProps {
   type: "text"
-  marks: Marks[]
+  marks?: Marks[]
   text: string
```
Comment (alexanderleegs) [inline src/interfaces/native/Text.ts:47]: Assuming this is another separate fix that's supposed to go in!

**Review by alexanderleegs — COMMENTED** [review]
Left some minor comments!

File: src/interfaces/native/Text.ts (hunk 3 above)
Comment (dcshzj) [reply src/interfaces/native/Text.ts:47]: Oops yup, the marks was marked as required by accident but a normal text node may be a plain one with no marks being applied.

File: src/templates/next/components/internal/Notification/Notification.tsx (hunk 1 above)
Comment (dcshzj) [reply src/templates/next/components/internal/Notification/Notification.tsx:20]: Oh yeah this one was a separate fix for the paragraph one, missed this one when splitting up the original PR.

File: src/templates/next/components/native/Table/Table.tsx (hunk 2 above)
Comment (dcshzj) [reply src/templates/next/components/native/Table/Table.tsx]: Added in 39b45ee6bd712996fa47a82207a8789dba0ce2b2!

**Review by alexanderleegs — APPROVED** [review]
lgtm

**Pull request merged**